### PR TITLE
Support change the rateLimiter of work queue from cli

### PR DESCRIPTION
**Pull request by cwdsuzhou**
See https://github.com/virtual-kubelet/virtual-kubelet/issues/840

Add rateLimiter to options, so we can use it by `cli.WithBaseOpts`

ping @cpuguy83  for review

### CODE REVIEW

**Comment by cwdsuzhou** [comment]
`Lint` prevents this PR, but I can not find the reasons. @cpuguy83 can you help me ?

File: go.mod [hunk 1]
```diff
@@ -3,60 +3,80 @@ module github.com/virtual-kubelet/node-cli
...
+	k8s.io/klog v1.0.0
+	k8s.io/kubernetes v1.17.6
+	k8s.io/kubectl v0.17.6
 )
 
-replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.0.0-20190805144654-3d5bf3a310c1
+replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.6
 
-replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.0.0-20190805144409-8484242760e7
+replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.6
```
Comment (cpuguy83) [inline go.mod:44]: If we update these, I think we can remove these `repalce` statements and put them inline with the main require statement instead.

**Comment by cpuguy83** [comment]
It's timing out for some reason.

**Comment by cpuguy83** [comment]
Possibly we should `go get` all the modules before running the linter.

**Comment by cpuguy83** [comment]
Nope, doesn't help.

**Comment by cpuguy83** [comment]
Fixed by pre-compiling packages before linting.

**Pull request merged**